public/scss/components/_nav.scss

```diff
@@ -28,7 +28,7 @@
   }
 
   &.-pinned {
-    padding: 0.5rem;
+    padding: 0.5rem 1rem;
```

Comment:
Miscellaneous fix unrelated to the main PR, the side spacing was getting a bit smooshed on the pinned nav variant.